messaging/src/test/java/org/axonframework/deadline/DefaultDeadlineManagerSpanFactoryTest.java

```diff
@@ -0,0 +1,143 @@
+                expectedSpan("DeadlineManager.cancelDeadline(myDeadline)", TestSpanFactory.TestSpanType.INTERNAL)
+                        .expectAttribute("this-is-a-stupidly-long-attribute-name", "myDeadlineId")
+        );
+    }
+
+    @Test
+    void createCancelAllSpanWithDefaultSettings() {
+        test(
+                spanFactory -> spanFactory.createCancelAllSpan("myDeadline"),
+                expectedSpan("DeadlineManager.cancelAllDeadlines(myDeadline)", TestSpanFactory.TestSpanType.INTERNAL)
+        );
+    }
+
+
+    @Test
+    void createCancelAllWithinScopeSpanWithDefaultSettings() {
+        ScopeDescriptor scope = Mockito.mock(ScopeDescriptor.class);
+        Mockito.when(scope.scopeDescription()).thenReturn("myScopeDescription");
+        test(
+                spanFactory -> spanFactory.createCancelAllWithinScopeSpan("myDeadline", scope),
+                expectedSpan("DeadlineManager.cancelAllWithinScope(myDeadline)", TestSpanFactory.TestSpanType.INTERNAL)
+                        .expectAttribute("axon.deadlineScope", "myScopeDescription")
+        );
+    }
+    @Test
+    void createCancelAllWithinScopeSpanWithModifiedScopeAttribute() {
+        ScopeDescriptor scope = Mockito.mock(ScopeDescriptor.class);
+        Mockito.when(scope.scopeDescription()).thenReturn("myScopeDescription");
+        test(
+                builder -> builder.scopeAttribute("this-is-a-stupidly-long-attribute-name"),
+                spanFactory -> spanFactory.createCancelAllWithinScopeSpan("myDeadline", scope),
+                expectedSpan("DeadlineManager.cancelAllWithinScope(myDeadline)", TestSpanFactory.TestSpanType.INTERNAL)
+                        .expectAttribute("this-is-a-stupidly-long-attribute-name", "myScopeDescription")
+        );
+    }
+
+    @Test
+    void createExecuteSpanWithDefaultSettings() {
+        DeadlineMessage<?> message = Mockito.mock(DeadlineMessage.class);
+        test(spanFactory -> spanFactory.createExecuteSpan("myDeadline", "myDeadlineId", message),
+                expectedSpan("DeadlineManager.executeDeadline(myDeadline)", TestSpanFactory.TestSpanType.HANDLER_LINK)
+                        .withMessage(message)
+                        .expectAttribute("axon.deadlineId", "myDeadlineId")
+        );
+    }
+
+    @Test
+    void createExecuteSpanWithModifiedDeadlineIdAtrribute() {
+        DeadlineMessage<?> message = Mockito.mock(DeadlineMessage.class);
+        test(
+                builder -> builder.deadlineIdAttribute("this-is-a-stupidly-long-attribute-name"),
+                spanFactory -> spanFactory.createExecuteSpan("myDeadline", "myDeadlineId", message),
+                expectedSpan("DeadlineManager.executeDeadline(myDeadline)", TestSpanFactory.TestSpanType.HANDLER_LINK)
+                        .withMessage(message)
+                        .expectAttribute("this-is-a-stupidly-long-attribute-name", "myDeadlineId")
+        );
+    }
+
+    @Test
+    void testPropagateContext() {
```

Comment:
Nit:
```suggestion
    void propagateContext() {
```

Reply:
Ahh yeah. Changed for the other instances too